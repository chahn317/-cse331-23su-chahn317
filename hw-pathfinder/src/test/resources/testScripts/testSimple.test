# Test a simple graph

CreateGraph graph1

AddNode graph1 A
AddNode graph1 B
AddNode graph1 C

AddEdge graph1 A B 1.0
AddEdge graph1 B C 1.0

FindPath graph1 A C