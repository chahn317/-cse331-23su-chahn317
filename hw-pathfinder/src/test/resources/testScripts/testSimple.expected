# Test a simple graph

created graph graph1

added node A to graph1
added node B to graph1
added node C to graph1

added edge 1.000 from A to B in graph1
added edge 1.000 from B to C in graph1

path from A to C:
A to B with weight 1.000
B to C with weight 1.000
total cost: 2.000